sql.txt

```diff
@@ -1,15 +1,54 @@
-(1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
-Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+(1) Using the Customers Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
+Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below.
+
+
+SELECT CustomerName, ContactName, Country 
+FROM Customers 
+ORDER BY Country 
+LIMIT 10 
+OFFSET 10;
+
+
+
+(2) Select all columns from the Customers table where the ContactName starts with A.  Paste your SQL statement below.
+
+
+SELECT * 
+FROM Customers 
+WHERE ContactName 
+LIKE "A%";
 
-(2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+
+SELECT * 
+FROM OrderDetails 
+WHERE ProductID = 51 AND Quantity > 10;
+
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+
+INSERT INTO Products VALUES
+	(NULL, 'Product 1', 2, 3, '10 boxes x 10 bags', 20),
+    	(NULL, 'Product 2', 3, 1, '10 boxes x 20 bags', 38),
+    	(NULL, 'Product 3', 4, 2, '10 boxes x 30 bags', 45);
+
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Just specify the new value; do 
 not try to get SQL to do the addition for you.)  Paste your SQL statement below
 
+
+UPDATE Products 
+SET Price = Price + 1.5 
+WHERE ProductID BETWEEN 1 AND 2;
```

Comment:
The exercise says we have to specify the value, not adding 1.50 using addition.

Reply:
@andresochoa91, yes you right, but John McGarvey slaked in sunrise-piranha channel and said that there is a bug in this task, so I can't do updating two rows in one statement without having SQL do the addition for me.

Reply:
Oh, okay, then, no worries :+1: